<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphologyTransformation</class>
 <widget class="QWidget" name="MorphologyTransformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="imageScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>609</width>
          <height>319</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="img">
           <property name="text">
            <string>Image</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Select Operation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="operationComboLabel">
              <property name="text">
               <string>Operation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="operationComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Opening</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Closing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Morphological Gradient</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Top Hat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Black Hat</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="scaleCheckBox">
            <property name="text">
             <string>Auto Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="structElmLabel">
            <property name="text">
             <string>Structuring Element:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="structElmComboBox">
            <item>
             <property name="text">
              <string>Rectangular</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Elliptic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cross Shaped</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="kSizeLabel">
         <property name="text">
          <string>Kernel Size (2n + 1), max = 21)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="kSizeSlider">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>21</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
